### PR TITLE
Update conversions_dashboard_v2.md

### PR DESCRIPTION
**Pull request by gsinghbraze**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that you haven't removed any images (replacing an image with an updated one of the same name is fine), as this breaks the French site
- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @josh-mccrowell-braze and @bre-fitzgerald as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [ ] If the documentation involves a 1) paid SKU, 2) a third party, 3) SMS, 4) AI, or 5) privacy, ensure that Maria Maldonado on the Legal team has signed off.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @josh-mccrowell-braze or @bre-fitzgerald for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@josh-mccrowell-braze</td>
    <td>Monolith Deployments<br>Quality Infrastructure<br>Platform Infrastructure<br>Datalake<br>SDKs<br>Currents</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Intelligence<br>In-App Messages<br>Channels<br>FIX</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Ingestion<br>Core Objects<br>Core Messaging<br>Messaging Experience<br>Message Components<br>Email (Composition and Infrastructure)</td>
  </tr>
  <tr>
    <td>@rachel-feinberg</td>
    <td>Customer Lifecycle, Identity and Permissions<br>SMS<br>User Targeting<br>Reporting</td>
  </tr>
</table>
</details>

### CODE REVIEW

**Comment by bre-fitzgerald** [comment]
@cla-bot check

File: _docs/_hidden/private_betas/conversions_dashboard_v2.md [hunk 1]
```diff
@@ -52,12 +58,17 @@ Your report is split into three sections:
...
 
-- *Unique Recipients* is defined as the number of users who received a message through the selected channel within the report's date range.
-- *Conversions* are defined by the attribution method you selected when you set up the report. See the following sections for additional details on the [attribution methods](#attribution-methods).
+| Column in table | Metric shown | Description |
+| --- | --- | --- |
+| 1 | Recipients | The number of users who received a message through the selected channel within the report's date range |
+| 2 | Conversion Rate (Recipients) | Calculated as: (Number of conversions) / (Number of recipients) |
+| 3 | Attribution method | Defined by the [attribution method](#attribution-methods) you selected when you set up the report. For Last Touch attribution or if multiple channels are selected, this appears as [Touches](#terms-to-know). |
+| 4 | Conversion Rate (Attribution method) | Defined by the [attribution method](#attribution-methods) you selected when you set up the report. If multiple channels are selected, this defaults to last-touch attribution. |
+{: .reset-td-br-1 .reset-td-br-2 .reset-td-br-3}
```
Comment (bre-fitzgerald) [inline _docs/_hidden/private_betas/conversions_dashboard_v2.md:69]:
```suggestion
| Metric shown | Description |
 | --- | --- |
| Recipients | The number of users who received a message through the selected channel within the report's date range |
| Conversion Rate (Recipients) | Calculated as: (Number of conversions) / (Number of recipients) |
| Attribution method | Defined by the [attribution method](#attribution-methods) you selected when you set up the report. For Last Touch attribution or if multiple channels are selected, this appears as [Touches](#terms-to-know). |
| Conversion Rate (Attribution method) | Defined by the [attribution method](#attribution-methods) you selected when you set up the report. If multiple channels are selected, this defaults to last-touch attribution. |
{: .reset-td-br-1 .reset-td-br-2 }
```

File: _docs/_hidden/private_betas/conversions_dashboard_v2.md [hunk 2]
```diff
@@ -8,24 +8,30 @@ description: "The conversions dashboard allows you to analyze conversions across
...
 ## Setting up your report
 
+To set up your conversions dashboard report:
+
 1. Navigate to **Analytics** > **Conversions**.
 2. Select a **Date Range** for your report, up to a 90-day window.
 3. Select the campaigns or Canvases (or both) that you would like to analyze. 
    - If you would like to filter campaigns and Canvases by tag, select a **Tag**.  
-4. Select a **Channel** that you would like to analyze for your messages. For now, you can select a single channel. 
+4. Select which **Channel(s)** you would like to analyze for your messages.
```
Comment (lydia-xie) [inline _docs/_hidden/private_betas/conversions_dashboard_v2.md:25]:
```suggestion
4. Select the **Channel(s)** you would like to analyze for your messages.
```

File: _docs/_hidden/private_betas/conversions_dashboard_v2.md [hunk 3]
```diff
@@ -69,6 +80,10 @@ By default, all selected campaigns and Canvases are shown. To deselect a campaig
 
 To download the time series data, click and select your download option. Available options are PNG, JPEG, PDF, SVG, or CSV.
 
+{% alert note %}
+This graph only shows data for a single channel at a time, please use the **Channel** dropdown on the chart to select a single channel.
```
Comment (lydia-xie) [inline _docs/_hidden/private_betas/conversions_dashboard_v2.md:84]:
```suggestion
This graph only shows data for a single channel at a time. Use the **Channel** dropdown on the chart to select a single channel.
```

**Review by lydia-xie — CHANGES_REQUESTED** [review]
lgtm - small suggestions for your review

**Review by lydia-xie — APPROVED** [review]

**Pull request merged**